packages/terra-consumer-layout/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ChangeLog
 
+# Unreleased
+
+### Changed
+- Fixed nav height in IE
```

Comment:
you can just add this to the beta.2 tag since it hasn't been released yet.
https://www.npmjs.com/package/terra-consumer-layout